tests/sentry/workflow_engine/endpoints/validators/test_base_workflow.py

```diff
@@ -225,3 +236,329 @@ def test_create__with_actions__creates_action_group(self, mock_action_handler):
         # check the action / condition group
         assert action_group.action.type == Action.Type.SLACK
         assert action_group.condition_group.logic_type == "any"
+
+
+class TestWorkflowValidatorUpdate(TestCase):
+    def setUp(self):
+        self.context = {
+            "organization": self.organization,
+            "request": self.make_request(),
+        }
+
+        self.action_filters = [
+            {
+                "actions": [
+                    {
+                        "type": Action.Type.SLACK,
+                        "config": {
+                            "target_identifier": "foo",
+                            "target_display": "bar",
+                            "target_type": 0,
+                        },
+                        "data": {},
+                        "integrationId": 1,
+                    }
+                ],
+                "logicType": "any",
+                "conditions": [],
+                "organizationId": self.organization.id,
+            }
+        ]
+
+        self.valid_data = {
+            "name": "test",
+            "enabled": True,
+            "actionFilters": self.action_filters,
+            "config": {
+                "frequency": 30,
+            },
+            "triggers": {
+                "logicType": "any",
+                "conditions": [
+                    {
+                        "type": Condition.EQUAL,
+                        "comparison": 1,
+                        "condition_result": True,
+                    },
+                ],
+            },
+        }
+
+        validator = WorkflowValidator(
+            data=self.valid_data,
+            context=self.context,
+        )
+
+        validator.is_valid(raise_exception=True)
+        self.workflow = validator.create(validator.validated_data)
+        self.context["workflow"] = self.workflow
+
+        serializer = WorkflowSerializer()
+        attrs = serializer.get_attrs([self.workflow], self.user)
+        self.valid_saved_data = serializer.serialize(self.workflow, attrs[self.workflow], self.user)
+
+    def test_update_property(self):
+        self.valid_data["name"] = "Update Test"
+        validator = WorkflowValidator(data=self.valid_data, context=self.context)
+
+        assert validator.is_valid() is True
+        workflow = validator.update(self.workflow, validator.validated_data)
+
+        assert workflow.id == self.workflow.id
+        assert workflow.name == "Update Test"
+
+    def test_update__remove_triggers(self):
```

Comment:
nit: maybe call this remove conditions? might be nice to assert the conditions count before updating too to make it more clear, took me a bit to understand what this test was doing